workflow/hpxml_inputs.json

```diff
@@ -516,7 +515,6 @@
     "water_heater_heating_capacity": 18767,
     "water_heater_jacket_rvalue": 0,
     "water_heater_setpoint_temperature": 125,
-    "water_heater_num_units_served": 1,
```

Comment:
Removing this now optional argument because this would now mean a shared system for 0 bedroom units.